File: Python/stack.py
class Stack:
    def __init__(self,stackElements):
        self.stackElements = stackElements

    def push(self):
        askElement = int(input("How  many elemnents you want to insert in Stack?"))
        while len(self.stackElements) < askElement:
            addElement = int(input("Enter the elements you want to insert in Stack?"))
            self.stackElements.append(addElement)
            print(self.stackElements)
    def sort(self):
        for  i in range(len(self.stackElements)):
            for j in range(i+1, len(self.stackElements)):
                if self.stackElements[i] > self.stackElements[j]:
                    self.stackElements[i] , self.stackElements[j] = self.stackElements[j] , self.stackElements[i]
        print(self.stackElements)
    def search(self):
        number = int(input('Enter the number you want to search? = '))
        if number in self.stackElements:

            print("Your searched number" ,number ,'is existed in stack.')
        else:
            print("The number you asked is not existed in the stack")

    def Pop(self):
            

            print(self.stackElements.pop())
            print(self.stackElements)

    def Insert(self):
        insertNumber = int(input("Enter the number you want to insert in the stack?"))
        definePosition = int(input("Define the position where you want to insert the number?"))
        if insertNumber != None and definePosition != None:
            self.stackElements.insert(definePosition,insertNumber)

            print(self.stackElements)
    
    



s = Stack([])
s.push()
s.sort()
s.search()
s.Pop()
s.Insert()

File: js/stack.js

//Adding or Pushing in Stack
let prompt = require("prompt-sync")({sigint:true});
let stackList = []
let inputNumbers =  prompt("How many elements you want to insert in the stack? = ")
while( stackList.length < inputNumbers ){
  ask = prompt("Enter the number you want to insert? = ");
  stackList.unshift(ask);
  
}

//Sorting stack

let numbers= stackList;
sortNumbers = numbers.sort(function(a, b){return a - b})
console.log("The sorted stack would be" + " " + sortNumbers)

//Searching
var askValue = prompt("Enter your required value to search? = ")
if (stackList.includes(askValue)){
  console.log("Your searched value is existed in stack")
} else{
  console.log("Your searched Value is not  existed in stack")
}
//Removing
var removeValue = prompt("Enter the value which you want to remove? = ");
for ( var i=0 ; i< stackList.length;  i++){
    if (stackList[i] === removeValue ){
      stackList.splice(i,1);
      i--;
      console.log(stackList)
    }
  }

//Pop from stack

stackList.pop()
console.log(stackList)

// insert in Stack
let position = prompt("Enter at which position you want to  insert the number? = ")
let insertNumber = prompt("Enter which number you want to insert? = ")
stackList.splice(position,0,insertNumber)
console.log(stackList)

